HandHistories.Parser/Parsers/FastParser/GGPoker/GGPokerFastParserImpl.cs

```diff
@@ -359,8 +354,19 @@ protected override SeatType ParseSeatType(string[] handLines)
 
         protected override GameType ParseGameType(string[] handLines)
         {
+            var line = handLines[0];
+            var colonIndex = line.IndexOf(':', 12);
+
+            // can be either 1 or 2 spaces after the colon
+            var startIndex = colonIndex + 1;
+            var endIndex = line.IndexOf('(', colonIndex) - 2;
```

Comment:
Please include the test in https://github.com/ItalyToast/PokerHandHistoryParser/pull/96

I found that GGPoker includes two versions of the hand history, the older one with two spaces in front of the limit string, while the older one with only one space before. See the unit tests in that pr for example.

This function is failing for the newer version.

Reply:
I've added the unittest